mautic/mautic
# [UX] Change “Publicly-visible Segment” option to “Visible to other users” since the actual label could lead users to think it’s visible to people outside of Mautic (leads)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The current label “Publicly-visible Segment” could be enhanced for better clarity. It would be advantageous to use a label that accurately reflects the visibility scope within the platform. The proposed solution in this pull request is to change the label to “Visible to other users”, which directly indicates that the segment is visible to other users of the platform, not to people outside of it.

This change is beneficial for users who are managing segments and could benefit from clearer visibility settings. By implementing this change, we can expect a more user-friendly experience when managing segment visibility, potentially reducing any confusion about who can see the segments. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Segments
3. Create a new segment
4. Look at the options on the right side of the screen 

![image](https://github.com/mautic/mautic/assets/116097999/bc6c3d5d-5e0f-40e3-a404-068d5465b2ee)
